slack_rules/slack_application_dos.yml

```diff
@@ -0,0 +1,85 @@
+AnalysisType: rule
+Filename: slack_application_dos.py
+RuleID: Slack.AuditLogs.ApplicationDoS
+DisplayName: Slack Denial of Service
+Enabled: true
+LogTypes:
+  - Slack.AuditLogs
+Tags:
+  - Slack
+Severity: Critical
+Description: Detects when user sessions are constantly reset in a way that makes Slack unusable
```

Comment:
suggested alt description "Detects when slack admin invalidates user session(s) more than once in a 24 hour period which can lead to DoS"